metrics_ci_integration/test/common/authorization/authorization_base_test.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:ci_integration/common/authorization/authorization.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('AuthorizationBase', () {
+    const header = 'header';
+    const token = 'token';
+
+    test(
+      'toMap() should include provided header and token as a key and '
+      'value respectively',
+      () {
+        final authorization = AuthorizationBaseTestbed(header, token);
+        final expected = {header: token};
+        final actual = authorization.toMap();
+
+        expect(actual, equals(expected));
+      },
+    );
+
+    test(
+      'should throw ArgumentError creating a new instance with null header',
```

Comment:
@vova-beloded @vladlaba  I think we need to decide about test description style. Should we make it a regular sentence or it should be informal like here:
```suggestion
      'should throw ArgumentError creating a new instance with null header',
```

or 
```suggestion
      'Should throw ArgumentError creating a new instance with null header.',
```
I see it's different now across our monorepo so it's probably better to choose one option.